src/components/App.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+import CalculatorName from './CalculatorName';
+
+const App = () => (
+  <div className="calculator">
+    <CalculatorName />
```

Comment:
Accordingly, to the project requirement, we don't need `CalculatorName` component.
kindly remove it.